css/index.css

```diff
@@ -57,15 +57,21 @@ html, body {
     font-family: 'Titillium Web', sans-serif;
 }
 
+html {
+    font-size: 62.5%;
+}
+
 h1, h2, h3, h4, h5 {
     font-family: 'Bangers', cursive;
     letter-spacing: 1px;
-    margin-bottom: 10px;
+    margin-bottom: .3%;
```

Comment:
As mentioned in yesterdays standup/solution meeting, top and bottom margins should be in px vs %. They are more for aesthetic properties rather than responsiveness 😄 But on saying that, the fact that you were able to get it to be responsive is awesome!